scripts/chroot_reconst.sh

```diff
@@ -70,10 +70,6 @@ echo "deb https://download.sublimetext.com/ apt/stable/" | tee /etc/apt/sources.
 apt-get -qq update
 apt-get -qq -y install sublime-text
 
-# Installing related softwares
```

Comment:
why are you removing this?

Reply:
I have not removed it just executed in the bottom of the script.